c++/README.md

```diff
@@ -10,24 +10,24 @@ bazel build :client :account-server :wallet-server :stats-server
 Run the account server:
```

Comment:
The bazel build command needs to install bazel first. Any easy way to do that?

And why does the `./tools/bazel` in grpc/grpc work without installing bazel?

Reply:
It's not hard to install bazel, in gprc/grpc, we also have instructions like :
***
Bazel is the primary build system for gRPC C++ and if you're comfortable with using bazel, we can certainly recommend it. Using bazel will give you the best developer experience as well as faster and cleaner builds.

You'll need bazel version 1.0.0 or higher to build gRPC. See Installing Bazel for instructions how to install bazel on your system. We support building with bazel on Linux, MacOS and Windows.

From the grpc repository root

# Build gRPC C++
$ bazel build :all
***

./tools/bazel actually will just do the installation of the latest version for you and then run it.   It does the installing for you.

Reply:
Let's make a copy of the bazel script?

Reply:
it didn't work directly, i need to make modifications.  Prob not that hard but let's do that separately?  The PR is already polished

Reply:
works now!  added tools/bazel